NOTICE

```diff
@@ -18,3 +18,4 @@ Copyright (C) 2023-2024 Double Open Oy
 Copyright (C) 2024 Robert Bosch GmbH
 Copyright (C) 2024 Cariad SE
 Copyright (C) 2025 Quartett mobile GmbH
+Copyright (C) 2025 Frank Viernau
```

Comment:
Hmm. So far @oss-review-toolkit/tsc did not add themselves under their personal email addresses, although I believe all of us did commits in their spare time and under a personal email address.

So I wonder whether we should keep things simple and not mention any individuals here, but refer to the output of `git shortlog -sne` instead? Otherwise we'd have to add quite a few more entries here for consistency.

Reply:
I'll close this for now.